ui/marketplace/useMarketplaceApps.tsx

```diff
@@ -52,8 +51,17 @@ export default function useMarketplaceApps(filter: string, selectedCategoryId: s
   const { value: isExperiment } = useFeatureValue('marketplace_exp', false);
 
   const { isPlaceholderData, isError, error, data } = useQuery<unknown, ResourceError<unknown>, Array<MarketplaceAppOverview>>({
-    queryKey: [ 'marketplace-apps' ],
-    queryFn: async() => apiFetch(configUrl, undefined, { resource: 'marketplace-apps' }),
+    queryKey: [ 'marketplace-dapps' ],
+    queryFn: async() => {
+      if (!feature.isEnabled) {
+        return [];
+      }
+      const url = 'configUrl' in feature ?
```

Comment:
I believe it is better to register new resources in the `lib/api/resources.ts` for admin API, and then either call a regular fetch or an API fetch:


```js
const fetch = useFetch();
const apiFetch = useApiFetch();
...
return feature.configUrl ? fetch(feature.configUrl): apiFetch('marketplace_dapps');
...
```